libraries/ui-library/src/components/six-select/six-select.tsx

```diff
@@ -217,6 +217,7 @@ export class SixSelect {
   }
 
   componentWillLoad() {
+    this.handleValueChange();
```

Comment:
This can likely be changed to:
```ts
componentWillLoad() {
  this.value = convertToValidValue(this.value, this.multiple);
  this.handleSlotChange();
}
```